### DISCUSSION TITLE
@check/verbose toggle (Or equivalent)

### DISCUSSION THREAD

**Discussion by somethingfinite** [Arx-Game/arxcode#379] [Ideas]
I'd love to have the option for a verbose output for rolls that outputs the number. Currently, accepting pre-rolls for contests, etc is impossible due to the fact that check/contest doesn't show the roll values it arrived at, it just gives a binary 'Xyz is the Winner!'

It would be nice if this were an opt-in toggle in the 'settings' area that defaulted to off so it's available as a level of granularity for those who care, but not in the way for people who really just care about their IC result.

Basically, just appending the rolled number in parenthetical.

Eg: '<Default Dicestring> Hellfrog checks mana and **METEORS** at daunting. Critical Success! Hellfrog is inhumanly successful in a way that defies expectations. (244)'

**Comment by TehomCD** [comment]
One thing I should mention is that for everything outside of breaking ties, result levels are the same - the number used to generate them isn't used. In other words, whether you rolled a 50 or 60 doesn't matter in any way if it generates the same result. I feel that having people display numbers could be somewhat misleading on that basis, and it might be better to have a more focused usability fix for contested rolls.

As I've mentioned to others, I'm not super opposed to transparency, but I'd rather it be less subject to interpretation: people seeing a raw number doesn't tell them as much as seeing exact percentages to succeed a roll, for example. I think in many cases throwing numbers at people devoid of context may just cause confusion.

**Reply by somethingfinite** [reply]
I think that particularly in the case of accepting pre-rolls (long a staple of contests), it's functionality that is currently missing that was there before. When oldcheck gets shuffled off, there won't be any means to take its place.

I rather disagree that showing numbers is confusing if it's an optional output. Very much a 'if I want this, I have it. If I can't be bothered, no thanks.' - I suppose I don't see the harm or confusion in something that has to be toggled on manually.

I have absolutely no desire to remove the descriptive text. It's clean and it's neat, and it's to the point. But there's some functionality that's being lost, at present without a way to gather the number.
